.circleci/test-api.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
+apt-get update
+apt-get install -y jq
+
+make serve-local &
+
+curl --retry-connrefused \
+    --retry 5 \
+    --retry-delay 20 \
+    http://127.0.0.1:5000 --fail
+
+var=$?
+
+if [[ $var -ne 0 ]]
+then
+  echo "Server failed to start."
```

Comment:
Is it really that the server failed to start, or that it is unreachable? 🙂 We don't know if it did start or not.